source/reference/beta-program.txt

```diff
@@ -25,6 +25,11 @@ Get Started
 To request access to the {+c2c-beta-program-short+} binaries, please file a 
 ticket with `support <https://support.mongodb.com/>`_.
 
+.. note:: 
+
+   Filing a support ticket does  *not* guarantee access to the 
+   {+c2c-beta-program-short+} binaries. 
```

Comment:
Can we qualify further to be "To request access to the mongosync beta binaries, please review the below disclaimer. Note that requests that do not conform to this disclaimer will **not** be approved. To continue with the request, please file a ticket with support."?

Reply:
I left out the mention of the disclaimer because in copy, we mention that users must accept the disclaimer when running the binary for the first time (which implies they already have access to the binary). In that case, should we remove the verbiage about the disclaimer popping up when running the binary?

Reply:
We have seen cases where customer hadn't reviewed the disclaimer before submitting the access request. My hope here is to 1) ask that they review the disclaimer before filing a ticket 2) also be aware that they need to explicitly accept the terms of the disclaimer when using the binary

Reply:
gotcha. updated